Sources/MapboxNavigation/Feedback/FeedbackType+FeedbackItem.swift

```diff
@@ -1,16 +1,10 @@
-import UIKit
 import MapboxCoreNavigation
-
-extension UIImage {
-    fileprivate class func feedbackImage(named: String) -> UIImage {
-        return Bundle.mapboxNavigation.image(named: named)!
-    }
-}
+import UIKit
 
 public extension FeedbackType {
 
     // TODO: Localize these strings
-    internal var title: String {
+    var title: String {
```

Comment:
Add docs

Reply:
It's still not public, just removed this `internal` keyword because it was redundant